src/components/CardList.js

```diff
@@ -1,7 +1,11 @@
 import ActivityCard from "./Card";
 import styled from "styled-components/macro";
+import Error from "./Error";
 
 function CardList({ activities, onJoin, onDeleteButtonClick }) {
+  if (!activities.length) {
+    return <Error />;
```

Comment:
Importing a component with another name than was defined looks quite confusing ;) 
Error is also to extreme. Call it Hint or Message

Reply:
Done!